specs/latest/2.0/index.html

```diff
@@ -699,12 +695,23 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  // WebGL1:
+  void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
+  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage);
+  void bufferSubData(GLenum target, GLintptr dstByteOffset, BufferDataSource? srcData);
```

Comment:
If you say so. I'm not 100% clear on Web IDL's overloading rules. I assume this has been checked with Firefox's IDL parser.